Makefile

```diff
@@ -7,20 +7,20 @@ all: test
 
 build:
 	@for dir in $(SUBDIRS) ; do \
-		$(MAKE) -C $$dir build || break; \
+		$(MAKE) -C $$dir build || exit 1; \
```

Comment:
Is the `exit 1` necessary? If you remove the `|| exit 1`, will it exit, or no?

Reply:
I tried that on my machine and it didn't stop execution.